interface_config.js

```diff
@@ -36,14 +36,14 @@ var interfaceConfig = { // eslint-disable-line no-unused-vars
      */
     TOOLBAR_BUTTONS: [
         //main toolbar
-        'microphone', 'camera', 'desktop', 'invite', 'fullscreen', 'hangup',
+        'microphone', 'camera', 'desktop', 'audioonly', 'invite', 'fullscreen', 'hangup', // jshint ignore:line
         //extended toolbar
         'profile', 'contacts', 'chat', 'recording', 'etherpad', 'sharedvideo', 'sip', 'dialpad', 'settings', 'raisehand', 'filmstrip'], // jshint ignore:line
     /**
      * Main Toolbar Buttons
      * All of them should be in TOOLBAR_BUTTONS
      */
-    MAIN_TOOLBAR_BUTTONS: ['microphone', 'camera', 'desktop', 'invite', 'fullscreen', 'hangup'], // jshint ignore:line
+    MAIN_TOOLBAR_BUTTONS: ['microphone', 'camera', 'desktop', 'audioonly', 'invite', 'fullscreen', 'hangup'], // jshint ignore:line
```

Comment:
Let's not add it to the main toolbar for now until we have a better idea of the UX we want.